priv/app/graph.jsx

```diff
@@ -152,17 +152,12 @@ export default class Graph extends React.Component {
   }
 
   chartId() {
-    var arity = this.props.mfa[2];
+    var formatted_mfa = Utils.formatMFA(this.props.mfa);
 
-    // Guess what? Dots in module name (and Elixir modules contains it - "Elixir.Enum":map/2) are problematic for ID.
-    var safe_module = this.props.mfa[0].replace(/\./g, "-");
-    safe_module = safe_module.replace(/'/g, "");
-
-    // And "*" is not a valid character too for an ID.
-    if (arity === "*") {
-      arity = "x";
-    }
-
-    return `chart_${safe_module}_${this.props.mfa[1]}_${arity}`;
+    // Characters that have special meaning in CSS selectors are not safe in an ID.
+    // (eg ':', '.', '?', '*' or single quote itself)
+    // (A very problematic example: "'Elixir.List':'keymember?'/*")
+    return "chart_" + formatted_mfa.replace(/[^A-Za-z0-9_-]/g, "-");
```

Comment:
*Nitpicking*: `-` in pattern is superfluous (we will replace it anyway via `-`) and if required - should be escaped
.